src/env.rs

```diff
@@ -57,6 +57,15 @@ pub fn sub_remaining_point(env: &Env, points: u64) {
     }
 }
 
+/// Try to substract remaining point computing the points with a/b and ceiling
+/// the result.
+pub fn sub_remaining_points_with_ratio(env: &Env, a: usize, b: usize) {
+    match a.checked_div(b) {
```

Comment:
why not just multiply len() by some gas_per_byte constant ?

Reply:
https://github.com/massalabs/sc-runtime/pull/49#discussion_r776673812